go/logger/standard.go

```diff
@@ -337,10 +337,15 @@ func PickFirstError(errors ...error) error {
 }
 
 func (log *Standard) CloneWithAddedDepth(depth int) Logger {
-	clone := *log
+	clone := Standard{
+		filename:        log.filename,
+		module:          log.module,
+		externalHandler: log.externalHandler,
+	}
```

Comment:
i don't know what it's for...i just copied the struct without copying the mutex.